PiicoDev_Unified.py

```diff
@@ -150,3 +180,20 @@ def create_unified_i2c(bus=None, freq=None, sda=None, scl=None):
     else:
         i2c = I2CUnifiedMachine(bus=bus, freq=freq, sda=sda, scl=scl)
     return i2c
+
+
+def scan_sensors(I2CUnifiedOS):
+    scn_str = ""
+    amnt_sens = len(piicodev_addr)
+    scn_addr = I2CUnifiedOS.scan()
+    for addr_curr in scn_addr:
+        flag=False
+        i = 0
+        for name, addr in piicodev_addr.items():
```

Comment:
I'd suggest checking out unpacking an index using enumerate, fairly sure that you can call this to the same effect to save on some lines:

```python
for i, name, addr in enumerate(piicodev_addr.items()):
```